dask/dataframe/groupby.py

```diff
@@ -914,11 +934,25 @@ def _build_agg_args_simple(result_column, func, input_column, impl_pair):
     )
 
 
-def _build_agg_args_var(result_column, func, input_column):
+def _build_agg_args_var(result_column, func, func_args, func_kwargs, input_column):
     int_sum = _make_agg_id("sum", input_column)
     int_sum2 = _make_agg_id("sum2", input_column)
     int_count = _make_agg_id("count", input_column)
 
+    # we don't expect positional args here
+    if func_args:
+        raise TypeError(
+            f"aggregate function '{func}' got unexpected positional arguments {func_args}"
+        )
+
+    # and we only expect ddof=N in kwargs
+    expected_kwargs = {"ddof"}
+    unexpected_kwargs = set(func_kwargs.keys()) - expected_kwargs
```

Comment:
Minor comment: `dict` key views are `set`-like, so you should be able to drop the `set(...)` call

```suggestion
    unexpected_kwargs = func_kwargs.keys() - expected_kwargs
```